src/utils/config.ts

```diff
@@ -79,5 +80,6 @@ export function getConfig(): StorageConfigType {
     logflareEnabled: getOptionalConfigFromEnv('LOGFLARE_ENABLED') === 'true',
     logflareApiKey: getOptionalConfigFromEnv('LOGFLARE_API_KEY'),
     logflareSourceToken: getOptionalConfigFromEnv('LOGFLARE_SOURCE_TOKEN'),
+    forwardHeaders: getOptionalConfigFromEnv('FORWARD_HEADERS')
```

Comment:
I would make this env variable and config value more specific to postgrest, ex:

`postgrestForwardHeaders` => `POSTGREST_FORWARD_HEADERS`